### DISCUSSION TITLE
Github Copilot typo

### DISCUSSION THREAD

**Discussion by 99067718** [community/community#34294] [Copilot Conversations]
After I disabled the github copilot because it kept crashing my visual studio I realised there is a typo in the github copilot.
Its not much, but it looks more professional if you fix this typo.
![image](https://user-images.githubusercontent.com/89968831/192480909-77b2d455-e24e-47d6-a4a7-9b5c67ae9161.png)

**Comment by iamcjwilliams** [comment]
@99067718 Thank you for your feedback!

**Reply by 99067718** [reply]
no problem
